go-controller/cmd/ovn-k8s-overlay/app/common.go

```diff
@@ -0,0 +1,504 @@
+package app
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"html/template"
+	"io"
+	"net"
+	"net/url"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	gcfg "gopkg.in/gcfg.v1"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/openvswitch/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/openvswitch/ovn-kubernetes/go-controller/pkg/util"
+	"github.com/urfave/cli"
+)
+
+const (
+	// ConfigFilePath is the config file path for more fine grained control..
+	ConfigFilePath = "/etc/openvswitch/ovn_k8s.conf"
+)
+
+var cfg config.Config
+
+func fetchOVNNB(context *cli.Context) error {
+	// Open ovn_k8s config file.
+	conf, err := os.Open(ConfigFilePath)
```

Comment:
We may as well have a fetchConfig function called from initMaster. It is a little confusing to get all these values from a function named "fetchOVNNB"

Reply:
add fetchConfig function